views/index.ejs

```diff
@@ -47,6 +47,9 @@
             });
           ga('require', 'urlChangeTracker');
     </script>
+    <noscript>
+     Javascript is disabled. Please enable javascript and refresh the page.
```

Comment:
You probably could have styling this or at least pop it in a `<p>`. As well, think the messaging could be improved, perhaps include a link to "how to enable JavaScript" something like http://www.enable-javascript.com/ perhaps?